packages/composites/token-allocation/src/dataProvider.ts

```diff
@@ -1,14 +1,17 @@
 import { Requester } from '@chainlink/external-adapter'
 import { PriceAdapter, ResponsePayload } from './types'
 
-const getPrices = (apiConfig: any) => async (
+const getPrices = (apiConfig: any) => (jobRunID: string) => async (
```

Comment:
Heh, there are three layers to this function now...
It's getting a little complex, so try to figure out a way to simplify here.